package.json

```diff
@@ -6,17 +6,31 @@
     "url": "https://github.com/openfoodfoundation/openfoodnetwork"
   },
   "scripts": {
-    "storybook": "start-storybook"
+    "storybook": "start-storybook",
+    "build": "bin/webpack --node-env=production"
   },
   "devDependencies": {
+    "@rails/webpacker": "^6.0.0-beta.7",
```

Comment:
if we change the requirements in gemfile to 5.3.0 or 6.0 w may need to adapt this so that the webpacker versions match.

Reply:
Yes, but we don't want to change it ;)

Reply:
ok 👍